packages/ui-toolkit/src/anchor/index.js

```diff
@@ -9,17 +9,20 @@ const BaseLink = styled(({ component, children, ...rest }) =>
   React.createElement(component, rest, children)
 )`
   color: ${props => props.theme.primary};
+  -webkit-text-fill-color: ${props => props.theme.primary};
```

Comment:
You could use `currentcolor` to avoid to directly refer to the value

Reply:
That is an amazing idea !

It's also wayy easier to delete once the problem gets resolved .

You are the best !